requirements/base.in

```diff
@@ -5,11 +5,19 @@ python-dotenv
 python-dateutil
 pytz
 raven
+Sphinx
+sphinx-rtd-theme
 
-Django<2.1
+Django
 django-axes
 django-choices
 django-cors-middleware
 
 djangorestframework
-zds-schema==0.17.8
+django-extra-fields
+django-filter==2.0.0
```

Comment:
most of these deps don't need to be listed here because they're already deps of `gemma-zaken-common`, see: https://github.com/VNG-Realisatie/gemma-zaken-common/blob/master/setup.cfg#L33